apps/server/src/modules/authorization/authorization.service.ts

```diff
@@ -51,19 +51,14 @@ export class AuthorizationService {
 		entityId: EntityId,
 		context: AuthorizationContext
 	): Promise<boolean> {
-		// TODO: This try-catch-block should be removed. See ticket: https://ticketsystem.dbildungscloud.de/browse/BC-4023
-		try {
-			const [user, object] = await Promise.all([
-				this.getUserWithPermissions(userId),
-				this.loader.loadAuthorizableObject(entityName, entityId),
-			]);
-			const rule = this.ruleManager.selectRule(user, object, context);
-			const hasPermission = rule.hasPermission(user, object, context);
+		const [user, object] = await Promise.all([
```

Comment:
What has this todo with the current ticket? This can be effect mutliple places. This try catch is "maybe" part of the next authorization service refactoring, when we speak in refinment in bigger group before.

Reply:
You're right, it's too much for this PR. I'm gonna roll that back.